### ISSUE TITLE
Support for aarch64

### ISSUE THREAD

**Issue by Arsey** [nginx-proxy/nginx-proxy#1297]
I've tried nginx-proxy on Jetson Nano, where we have aarch64 and it doesn't work. Firstly I had to build the image manually but then when I start a container like `docker run --rm -it -p 80:80 -v /var/run/docker.sock:/tmp/docker.sock:ro jwilder/nginx-proxy-local` it ends up with
```
WARNING: /etc/nginx/dhparam/dhparam.pem was not found. A pre-generated dhparam.pem will be used for now while a new one
is being generated in the background.  Once the new dhparam.pem is in place, nginx will be reloaded.
/app/docker-entrypoint.sh: line 34: /usr/local/bin/forego: cannot execute binary file: Exec format error
/app/docker-entrypoint.sh: line 34: /usr/local/bin/forego: Success
```

**Comment by Knight1** [comment]
You have to download the forego arm version

**Comment by buchdag** [comment]
Fixed as of #1470 + 4a1d5bfc58af35f5e90dcefb47b35a274c903150 + d120ce9fdebf09a6c7b90ecff7eda0b7d2760ef3

`arm64` and `arm/v7` images are availables from `jwilder/nginx-proxy` or the new registry `nginxproxy/nginx-proxy`
